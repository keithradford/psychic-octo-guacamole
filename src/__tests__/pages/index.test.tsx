import "@testing-library/jest-dom";
import { render } from "@testing-library/react";
import Home from "../../pages";

describe("Index page", () => {
  it("should render correctly", () => {
    const { container } = render(<Home />);
    expect(container).toMatchSnapshot();
  });
});
