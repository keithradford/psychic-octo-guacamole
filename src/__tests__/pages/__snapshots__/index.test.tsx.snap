// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index page should render correctly 1`] = `
<div>
  <div>
    UVic Finance System
  </div>
</div>
`;
